exoscale/api/compute.py

```diff
@@ -1187,14 +1235,12 @@ def anti_affinity_groups(self):
             latency.
         """
 
-        try:
-            [res] = self.compute.cs.getInstancePool(
-                id=self.id, zoneid=self.zone.id, fetch_list=True
-            )
-            for aagid in res.get("affinitygroupids", []):
-                yield self.compute.get_anti_affinity_group(id=aagid)
-        except CloudStackApiException as e:
-            raise APIException(e.error["errortext"], e.error)
+        res = self.compute._v2_request(
+            "GET", "/instance-pool/" + self.id, self.zone.name
+        )
+
+        for i in res.get("anti-affinity-groups", []):
+            yield self.compute.get_anti_affinity_group(id=i["id"])
```

Comment:
same than for `instances`, why trying to return fully detailed information on a _list_ operation. It is not what was defined in public-api API